src/types/analytics.ts

```diff
@@ -1,6 +1,29 @@
-import { Message } from '../message/types'
+import {
+  Name as CommonEventNames,
+  AllTrackingEvents as CommonTrackingEvents
+} from 'audius-client/src/common/models/Analytics'
 
-type JsonMap = Record<string, unknown>
+import { Message } from 'app/message'
+
+export * from 'audius-client/src/common/models/Analytics'
```

Comment:
So this means we are export both `Name` and `EventNames` from here, that might be a bit confusing. Probably best not to re-export `Name`

Reply:
yeah good call, will explicitely export the needed items